programs/virtual-curve/src/instructions/partner/ix_create_partner_metadata.rs

```diff
@@ -0,0 +1,54 @@
+use crate::constants::seeds::PARTNER_METADATA_PREFIX;
+use crate::state::PartnerMetadata;
+use crate::EvtPartnerMetadata;
+use anchor_lang::prelude::*;
+
+#[derive(AnchorSerialize, AnchorDeserialize)]
+pub struct CreatePartnerMetadataParameters {
+    // padding for future use
+    pub padding: [u8; 48],
```

Comment:
Should be `[u8; 96]`? In case, we are gonna fully utilize the `PartnerMetadata::padding` ([u128; 6])

Reply:
Fixed